pystac/asset.py

```diff
@@ -261,6 +262,12 @@ def delete(self) -> None:
         href = _absolute_href(self.href, self.owner, "delete")
         os.remove(href)
 
+    @property
+    def ext(self) -> AssetExt:
```

Comment:
We should docstring these public properties. I'll just drop this one comment about that.